src/templates/partials/newVerificationCodeContent.mjml

```diff
@@ -0,0 +1,83 @@
+<mj-section css-class="card">
+  <mj-column>
+    <mj-text>
+      {{#if @root.type.register}}
+        ${{verificationCode.register.content_1}}$
+      {{/if}}
+
+      {{#if @root.type.emailOtp}}
+        ${{verificationCode.emailOtp.content_1}}$
+      {{/if}}
+
+      {{#if @root.type.emailVerify}}
+        ${{verificationCode.emailVerify.content_1}}$
+      {{/if}}
+    </mj-text>
+
+    <mj-spacer height="24px" />
+
+   
+    <!-- Button -->
+    <mj-button href="{{link}}" align="left" background-color="#0D6763" font-size="14px" font-weight="500" line-height="22px" inner-padding="12px 72px" color="white" border-radius="100px">
+      {{#if @root.type.register}}
+        ${{verificationCode.register.button_text}}$
+      {{/if}}
+
+      {{#if @root.type.emailOtp}}
+        ${{verificationCode.emailOtp.button_text}}$
+      {{/if}}
+
+      {{#if @root.type.emailVerify}}
+        ${{verificationCode.emailVerify.button_text}}$
+      {{/if}}
+    </mj-button>
+
+    <mj-spacer height="24px" />
+
+    <!-- Link / Code -->
+    <mj-raw>
+      {{#if @root.type.register}}
+    </mj-raw>
+      <mj-text>
+        ${{verificationCode.register.content_2}}$
+      </mj-text>
+    <mj-raw>
+      {{/if}}
+    </mj-raw>
+
+    <mj-raw>
+      {{#if @root.type.emailOtp}}
+    </mj-raw>
+      <mj-text>
+        ${{verificationCode.emailOtp.content_2}}$
+      </mj-text>
+      <mj-spacer height="24px" />
+      <mj-text>
+        ${{verificationCode.emailOtp.content_3}}$
+      </mj-text>
+      <mj-spacer height="8px" />
+      <mj-text font-size="14px" font-weight="600" color="#0D6763">
+        <span style="width: 240px;background:#F7F7F7;padding:12px 16px 12px 16px;border-radius:8px;display:inline-block;text-align:center"> {{@root.code}} </span>
```

Comment:
Inline styles may not work that well in some email clients, better to have a compatibity test.